library/pod-security-policy/fsgroup/template.yaml

```diff
@@ -3,7 +3,7 @@ kind: ConstraintTemplate
 metadata:
   name: k8spspfsgroup
   annotations:
-    description: Controls allocating an FSGroup that owns the Pod's volumes.
+    description: "Controls allocating an FSGroup that owns the Pod's volumes.  Corresponds to the `fsGroup` field in a PodSecurityPolicy."
```

Comment:
I wish there could be an "extended discription", so we can have a blurb and then more detailed instructions.

Reply:
Yea I was thinking about that.  Particularly something that would allow for some nice markdown syntax.

Reply:
Yeah, interesting to think how it would interact with shorter descriptions and examples...

File an issue for a feature request?

Reply:
Where does this feature request go?  Isn't this part of JSONSchema?  Or are you talking about extended description for the whole resource?  Couldn't we just add another annotation for that?

Reply:
The second (another annotation), just an FR against the library.

Reply:
Issue created: https://github.com/open-policy-agent/gatekeeper-library/issues/112